### PR TITLE
feature(lightbox): you can now resize the lightbox using the AMD module

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

**Comment by hypeJunction** [comment]
Me likes, but I it would make sense to allow width/height parameters:

> **$.colorbox.resize()** This allows Colorbox to be resized based on its own auto-calculations, or to a specific size. This must be called manually after Colorbox's content has loaded. The optional parameters object can accept width or innerWidth and height or innerHeight. Without specifying a width or height, Colorbox will attempt to recalculate the height of its current content.

File: views/default/elgg/lightbox.js [hunk 1]
```diff
@@ -114,6 +114,14 @@ define('elgg/lightbox', function (require) {
 		 */
 		close: function () {
 			$.colorbox.close();
+		},
+		
+		/**
+		 * Resizes the colorbox
+		 * @return void
+		 */
+		resize: function () {
```
Comment (mrclay) [inline views/default/elgg/lightbox.js]: I assume @hypeJunction is suggesting here:
```js
resize: $.colorbox.resize
```

Comment (mrclay) [inline views/default/elgg/lightbox.js]: I'd be OK adding that and squeezing it in 2.2. Better than folks continuing to call colorbox API directly.

Comment (hypeJunction) [inline views/default/elgg/lightbox.js]: I am on board with adding to 2.2

**Comment by hypeJunction** [comment]
Replaced by #9979

**Pull request closed without merging**